src/WorkflowStep.ts

```diff
@@ -24,23 +24,30 @@ export interface StepConfigureArguments {
 }
 
 export interface StepUpdateArguments {
-  inputs?: {};
-  outputs?: [];
+  inputs?: {
+    [key: string]: {
+      value: any;
```

Comment:
i think we can narrow this type to something more specific than `any`. it surely cannot be functions or other non-JSON types. @misscoded said she would figure out what the type here should be and offer a suggestion.

Reply:
According to the docs, this mirrors what's there, but it still gives me pause. How can a value be anything other than a `string`? Same for the key value in `variables`.

https://api.slack.com/reference/workflows/workflow_step#input

If you're confident in this @seratch, I'm ok with this change. I did leave one additional comment about `outputs` that follows the same line of thought.

Reply:
As far as I know, it can be `string | string[]` but the document clearly says it's "Any". https://api.slack.com/reference/workflows/workflow_step#input

This is not really an essential part of this PR. If we need more time to decide on this, I can revert this change for now. I wanted to fix the `StepCompleteArguments` issue promptly. It's not trivial for TypeScript users.

Reply:
I'm fine with it the way it is -- it's still an improvement from where it was and it does match the API docs at this time. We can always circle back and refine further at a later time.

Reply:
sounds good to me! i'll approve, merge, and do the release ASAP.